### PR TITLE
DEXP1586: Modified Account Registration Code

### PR DESCRIPTION
**Pull request by asmita32675**
https://rackspace.atlassian.net/browse/DEXP-1586

@ssalinas24 Please review.

### CODE REVIEW

File: content/how-to/account-management/account-registration-code/index.md [hunk 1]
```diff
@@ -1,26 +1,22 @@
 ---
 permalink: account-registration-code
```
Comment (ssalinas24) [inline content/how-to/account-management/account-registration-code/index.md:2]: This permalink and the account registration hyperlink take me to the page below.
Screen Shot 2022-09-26 at 9 39 34 PM

So it seems to be going to the other site. Look for a redirect and fix if needed.

File: content/how-to/account-management/account-registration-code/index.md [hunk 2]
```diff
@@ -1,26 +1,22 @@
...
-walks you through how to log in to the
-[Cloud Control Panel](https://login.rackspace.com)
-for the first time. Your first login is the only time that you need the
-registration code, unless you forget your user name.
+Rackspace uses an _account registration code_ to verify new users on an account, much like two-factor authentication as shown in below image. 
 
-The primary contact on the account is the only person who can change this
-code. When the primary contact adds a new user, the new user receives a
-registration code. The primary contact must give the new user their code.
+<img width="776" alt="accountregistrationcode1" src="/support/how-to/account-registration-code/accountregistrationcode1.png">
```
Comment (ssalinas24) [inline content/how-to/account-management/account-registration-code/index.md:14]: fix alt tag to be a few words that describe what the image is showing

File: content/how-to/account-management/account-registration-code/index.md [hunk 3]
```diff
@@ -1,26 +1,22 @@
...
 
-If you are the primary contact on the account and you forget the code and need
-to reset your login, contact your Account Manager or Rackspace Support.
+If an account administrator adds you as a new user, an account registration code is generated for your new user account and you receive an email notification that walks you through how to log in to the [Cloud Control Panel](https://login.rackspace.com) for the first time. Your **First Login** is the only time that you need the registration code, unless you forget your user name.
```
Comment (ssalinas24) [inline content/how-to/account-management/account-registration-code/index.md:16]:
```suggestion
If an account administrator adds you as a new user, an account registration code is generated for your new user account. You will receive an email notification that walks you through how to log in to the [Cloud Control Panel](https://login.rackspace.com) for the first time. Your **First Login** is the only time that you need the registration code, unless you forget your user name.
```

File: content/how-to/account-management/account-registration-code/index.md [hunk 4]
```diff
@@ -1,26 +1,22 @@
...
+
+<img width="510" alt="accountregistrationemail" src="/support/how-to/account-registration-code/accountregistrationemail.png">
```
Comment (ssalinas24) [inline content/how-to/account-management/account-registration-code/index.md:18]: fix alt tag

**Review by ssalinas24 — CHANGES_REQUESTED** [review]
this is a pretty quick PR to fix and review. So if you can fix, then we should be able to quickly re-review and merge. This is the beauty of having only one Jira for one PR. It keeps the changes small and quick to review.

File: content/how-to/account-management/account-registration-code/index.md (hunk 1 above)
Comment (asmita32675) [reply content/how-to/account-management/account-registration-code/index.md:2]: Need to discuss.

File: content/how-to/account-management/account-registration-code/index.md (hunk 2 above)
Comment (asmita32675) [reply content/how-to/account-management/account-registration-code/index.md:14]: Added "alphanumericcode"

File: content/how-to/account-management/account-registration-code/index.md (hunk 3 above)
Comment (asmita32675) [reply content/how-to/account-management/account-registration-code/index.md:16]: Updated

File: content/how-to/account-management/account-registration-code/index.md (hunk 4 above)
Comment (asmita32675) [reply content/how-to/account-management/account-registration-code/index.md:18]: added "receivedemail"

File: content/how-to/account-management/account-registration-code/index.md (hunk 1 above)
Comment (asmita32675) [reply content/how-to/account-management/account-registration-code/index.md:2]: Done

**Review by ssalinas24 — APPROVED** [review]

**Pull request merged**